WorkoutTest/DayViewController.swift

```diff
@@ -80,3 +89,23 @@ extension DayViewController: UITableViewDelegate {
     tableView.deselectRow(at: indexPath, animated: true)
   }
 }
+
+//MARK: - ShowActionSheet
+
+extension DayViewController {
+  
+  func showActionSheet() {
+    let alertController = UIAlertController(title: "Do you really want to remove this program?", message: nil, preferredStyle: .actionSheet)
+    
+    let deleteAction = UIAlertAction(title: "Delete", style: .destructive) { _ in
+      self.confirmDelete?()
+      self.navigationController?.popViewController(animated: true)
+    }
+    let cancelAlert = UIAlertAction(title: "Cancel", style: .cancel) { _ in
+    }
```

Comment:
Usually we make such code in one line  
```suggestion
    let cancelAlert = UIAlertAction(title: "Cancel", style: .cancel) { _ in  }
```